demos/basics/loading-pandas.ipynb

```diff
@@ -0,0 +1,3580 @@
+{
```

Comment:
<p>I think this link to the documentation may break if/when #1127 is merged?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>Yeah; I'll update the link on the assumption #1127 will merge soon given it's been approved.</p>